source/_integrations/aws.markdown

```diff
@@ -3,6 +3,7 @@ title: Amazon Web Services (AWS)
 description: Instructions on how to integrate Amazon Web Services with Home Assistant.
 ha_category:
   - Notifications
+ha_iot_class: Cloud Polling
```

Comment:
I believe this integration only pushes data to the cloud. Since there are no sensors, it isn't polling.

```suggestion
ha_iot_class: Cloud Push
```